src/compound/MorphoGovernance.sol

```diff
@@ -467,7 +472,7 @@ abstract contract MorphoGovernance is MorphoUtils {
         Types.MarketPauseStatus storage pause = marketPauseStatus[_poolToken];
 
         pause.isSupplyPaused = _isPaused;
-        pause.isBorrowPaused = _isPaused;
+        if (!pause.isDeprecated) pause.isBorrowPaused = _isPaused;
```

Comment:
```suggestion
        if (_isPaused || !pause.isDeprecated) pause.isBorrowPaused = _isPaused;
```

Reply:
Great catch ! So the idea would be to always allow to pause the markets, as it is already done in `setIsBorrowPaused`. The code is similar to the one of Morpho-AaveV3 in this case, so if we implement it here we should probably also implement it in Morpho-AaveV3

Reply:
I agree it would be better

Reply:
I think the current code and the suggest code are practically the same. If the market is deprecated, then we know that `isBorrowPaused` is already `true`. Because this is a prerequisite for calling `setIsDeprecated` and you can't set borrow paused to `false` until the market is un-deprecated, see `setIsBorrowPaused`.

The flow should be:
isBorrowPaused = true -> isDeprecated = true -> isDeprecated = false -> isBorrowPaused = false

So if `pause.IsDeprecated` then also `pause.isBorrowPaused` and allowing it to be overwritten by doing `|| _isPaused` doesn't really do much. Admittedly, the suggested code is easier to understand 😁

Reply:
I would either: 
- take the suggested code, as it is easier to understand
- or put a comment on why it it not useful to skip it if isPaused is true

Reply:
Ok I've been thinking about it and I'm joining @MathisGD on this. I don't think we should change the code

Reply:
So do we add a comment or not ?

Reply:
I can add one yes, open to suggestion

Reply:
Not mandatory though, the code is already clear
```suggestion
        // Note that pause.isDeprecated implies pause.isBorrowPaused.
        if (!pause.isDeprecated) pause.isBorrowPaused = _isPaused;
```